### PR TITLE
Tomes draw runes on targeted floors/walls, and wall talisman runes 

### PR DESCRIPTION
**Pull request by Fryer22**
Changes:
-Arcane Tomes now use a browser window popup.
-Words are found via normal means and entered into the notes section. One then sets the type of rune to draw (similar to crayons).
-The tome now draws a rune on whatever floor tile or wall you use it on. 

-To go along with this, wall talismans are added. Wall talismans work similar to paper talismans, but without the paper. A stun, blind, deaf, teleport, or EMP rune is drawn on a wall next to a Wall Imbue rune. Using the wall imbue rune deletes the imbued from rune and the wall imbue rune and creates a wall talisman in its place. The best way to describe the wall talismans shape is a directional window. When a person runs into it (specifically, the bumped proc) it activates. The "window" is flush against the wall, allowing people to navigate if they move slowly. Wall talismans cannot be drawn on the floor.

### CODE REVIEW

File: code/game/gamemodes/cult/ritual.dm [hunk 1]
```diff
@@ -146,11 +149,25 @@ var/engwords = list("travel", "blood", "join", "hell", "destroy", "technology",
...
+		if(desc && !usr.stat)
+			usr << "It reads: <i>[desc]</i>."
+			sleep(30)
+			explosion(src.loc, 0, 2, 5, 5)
+			qdel(src)
+		*/
+	if(!desc)
+		usr << "A spell circle drawn in blood. It reads: <i>[word1] [word2] [word3]</i>."
+	else
+		usr << "Explosive Runes inscription in blood. It reads: <i>[desc]</i>."
```
Comment (Anonus) [inline code/game/gamemodes/cult/ritual.dm]: Why this entire section? Why usr?

Comment (Fryer22) [inline code/game/gamemodes/cult/ritual.dm]: Not my doing, that is from how the code was when I started this. I'll fix that though. I've no idea whose idea it was to add explosions however.

**Comment by Jordie0608** [comment]
Changelog

**Comment by Cheridan** [comment]
This +nearly a thousand lines for a minor feature that I doubt would even see much use. Rune code simply isn't good enough to expand upon like this. Just making more work down the line.

**Pull request closed without merging**